forge-ai/src/main/java/forge/ai/AiController.java

```diff
@@ -262,17 +263,16 @@ public boolean checkETBEffects(final Card card, final SpellAbility sa, final Api
             }
         }
 
-        if (card.isCreature()
-                && game.getStaticEffects().getGlobalRuleChange(GlobalRuleChange.noCreatureETBTriggers)) {
-            return api == null;
-        }
-
         boolean rightapi = false;
 
         // Trigger play improvements
         for (final Trigger tr : card.getTriggers()) {
             // These triggers all care for ETB effects
 
+            if (StaticAbilityDisableTriggers.disabled(game, TriggerType.ChangesZone, tr)) {
```

Comment:
Shouldn't the below be enough?

Reply:
Oops, yes, meant for it to only be there once!